vendor.conf

```diff
@@ -46,6 +46,6 @@ github.com/etcd-io/bbolt v1.3.2
 github.com/klauspost/pgzip v1.2.1
 github.com/klauspost/compress v1.4.1
 github.com/klauspost/cpuid v1.2.0
-github.com/vbauerster/mpb v3.3.4
+github.com/vbauerster/mpb v3.4.0
```

Comment:
Is this change intentional?

Reply:
Yes, that version adds the `AddSpinner` call.